core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -337,6 +338,8 @@ object KafkaConfig {
   val SslClientAuthProp = SslConfigs.SSL_CLIENT_AUTH_CONFIG
 
   /** ********* SASL Configuration ****************/
+  val SaslMechanismProp = SaslConfigs.SASL_MECHANISM
```

Comment:
@ijuma Since the property is used by common code rather than broker-specific code, the same property is used for both clients and broker.